test/e2e/data/infrastructure-aws/kustomize_sources/machine-pool/spot-instance-machine-pool.yaml

```diff
@@ -24,7 +24,7 @@ kind: AWSMachinePool
 metadata:
   name: ${CLUSTER_NAME}-mp-1
 spec:
-  minSize: 1
+  minSize: 0
```

Comment:
Oh nice. :D :D So this prevented it to scale down to zero? :)

Reply:
Yes 😓